compiler/ppx/ppx_optcomp_light.ml

```diff
@@ -181,32 +179,37 @@ let rec filter_pattern = function
   | { ppat_attributes; ppat_loc; _ } as p ->
       if keep ppat_loc ppat_attributes then Some p else None
 
-let mapper =
-  { Ast_mapper.default_mapper with
-    cases =
-      (fun mapper cases ->
-        let cases =
```

Comment:
Can we maybe add support for this in ppxlib

Reply:
Yes, that's a good idea. Thanks! I've done that now.